src/main/java/thePackmaster/cardmodifiers/basicspack/DuplicateModifier.java

```diff
@@ -0,0 +1,59 @@
+package thePackmaster.cardmodifiers.basicspack;
+
+import basemod.abstracts.AbstractCardModifier;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardQueueItem;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.cardmodifiers.madsciencepack.AbstractMadScienceModifier;
+
+public class DuplicateModifier extends AbstractMadScienceModifier {
```

Comment:
Very similar to the Expand modifier, if you look at that you'll find this code
`GameActionManager.queueExtraCard(funCard, (AbstractMonster) target);`
Which is a little helper method that you could use